zkvm/docs/zkvm-spec.md

```diff
@@ -825,7 +843,7 @@ The ZkVM state consists of the static attributes and the state machine attribute
     * `tx_signature`
     * `cs_proof`
 2. Extension flag (boolean)
-3. Uniqueness flag (boolean)
+3. Last anchor (optional [anchor](#anchor): `None` or `Some(anchor)`)
```

Comment:
Avoid Rust idioms in the spec. I suggest something like:

```suggestion
3. Last [anchor](#anchor) or ∅ if unset.
```

Reply:
good point